.github/workflows/windows.yml

```diff
@@ -91,7 +91,11 @@ jobs:
       - name: pytest
         timeout-minutes: 10
         shell: bash -l {0}
+        env:
+          CONDA_OVERRIDE_CUDA: 11.8
```

Comment:
I think this needs to be in the `Mantid Imaging developer dependencies` step

Reply:
Actually I think you can do it at the job level https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow